tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -26,7 +26,6 @@ BASE_TARBALLS = serial-targets \
 	kernel-nvidia-gpu-confidential-tarball \
 	kernel-tarball \
 	nydus-tarball \
-	ovmf-sev-tarball \
```

Comment:
@arvindskumar99 , please see the discussion about the removal of ovmf-sev in https://github.com/kata-containers/kata-containers/pull/10920 . We should not remove it; also it would be good to use a name other than 'ovmf-sev'.

Reply:
Yeah please rename it. We wanted to remove it because of the name and not because of the function it provides.
```
ovmf-mdb -- mesaured-direct-boot
ovmf-kh -- kernel-hashes 
```
or even simply `ovmf-snp`

Reply:
@ryansavino FYI ^^ WDYT?